### ISSUE TITLE
FreeCad 1.0.0:   'NoneType' object has no attribute 'export'

### ISSUE THREAD

**Issue by AZtech** [kovacevic39/laser_post#3]
I have a fresh installation of FreeCAD (Flatpak)  installed on Linux Mint (21.3 Cinnamon). I copied the file "laser_post.py" to the macro directory. The post processor shows up in the list in Freecad as "laser" and it can be selected. It shows errors about missing tooltips:

```
09:57:30  pyException: Traceback (most recent call last):
  File "/app/freecad/Mod/CAM/Path/Main/Job.py", line 525, in onChanged
    self.tooltip = processor.tooltip
                   ^^^^^^^^^^^^^^^^^
<class 'AttributeError'>: 'NoneType' object has no attribute 'tooltip'
```

When I run the post processor the following error is thrown:

```
09:57:57  Running the Python command 'CAM_Post' failed:
Traceback (most recent call last):
  File "/app/freecad/Mod/CAM/Path/Post/Command.py", line 188, in Activated
    post_data = postprocessor.export()
                ^^^^^^^^^^^^^^^^^^^^

'NoneType' object has no attribute 'export'
```

No G-code file is created.

EDIT:
I installed the latest version of "laser_post" now it works. Thanks!

**Comment by kovacevic39** [comment]
I have fixed the compatibility issues with FreeCAD 1.0.0
Let me know if this works.

**Comment by AZtech** [comment]
Now it works. Thanks!
